### DISCUSSION TITLE
remove 'people' tab in the header

### DISCUSSION THREAD

**Discussion by jue0nkim** [alshedivat/al-folio#2701] [Q&A] [answered]
I would like to remove people tab in the header.
The people folder is in gh-pages branch, not in master, so I can't use the method that change nav: true to false.

How should I do?
Thank you

**Comment by xinyindeed** [comment]
I have the same question, there are several tabs I wouldn't need on my website. How should I go about removing them permanently?

**Comment by george-gca** [comment, marked as answer]
You should never touch the `gh-pages` branch.

Please take a look at our [customization guide](https://github.com/alshedivat/al-folio/blob/master/CUSTOMIZE.md), where the directories structures are explained. To remove the pages simply delete the files from `_pages/`.

Also similar to #2031, #2360, #627, #253.
